Makefile

```diff
@@ -172,6 +172,37 @@ ifeq ($(BUILD_ENTERPRISE_READY),true)
 	docker-compose -f docker-compose.yaml up haproxy
 endif
 
+run-multi: run-client
+ifeq ($(BUILD_ENTERPRISE_READY),true)
+	@echo Removing possibly wrong configured default network
+	docker network rm mattermost-server_mm-test
+
+	@echo Adding necessary host alias in /etc/hosts
+
+	./scripts/update_hosts
```

Comment:
What about Windows users?

Reply:
aha, good point will have to condition the script with some powershell